analyses/cell-type-consensus/references/consensus-cell-type-reference.tsv

```diff
@@ -0,0 +1,302 @@
+panglao_ontology	panglao_annotation	blueprint_ontology	blueprint_annotation_main	blueprint_annotation_fine	consensus_ontology	consensus_annotation
+CL:0000583	alveolar macrophage	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000767	basophil	CL:0000775	Neutrophils	Neutrophils	CL:0000094	granulocyte
+CL:0000771	eosinophil	CL:0000775	Neutrophils	Neutrophils	CL:0000094	granulocyte
+CL:0000235	macrophage	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000097	mast cell	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000576	monocyte	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000775	neutrophil	CL:0000775	Neutrophils	Neutrophils	CL:0000775	neutrophil
+CL:0000092	osteoclast	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000091	Kupffer cell	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000453	Langerhans cell	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000129	microglial cell	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000889	myeloid suppressor cell	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000874	splenic red pulp macrophage	CL:0000775	Neutrophils	Neutrophils	CL:0000766	myeloid leukocyte
+CL:0000767	basophil	CL:0000576	Monocytes	Monocytes	CL:0000766	myeloid leukocyte
+CL:0000451	dendritic cell	CL:0000576	Monocytes	Monocytes	CL:0000113	mononuclear phagocyte
+CL:0000771	eosinophil	CL:0000576	Monocytes	Monocytes	CL:0000766	myeloid leukocyte
+CL:0000097	mast cell	CL:0000576	Monocytes	Monocytes	CL:0000766	myeloid leukocyte
+CL:0000576	monocyte	CL:0000576	Monocytes	Monocytes	CL:0000576	monocyte
+CL:0000775	neutrophil	CL:0000576	Monocytes	Monocytes	CL:0000766	myeloid leukocyte
+CL:0000784	plasmacytoid dendritic cell	CL:0000576	Monocytes	Monocytes	CL:0000113	mononuclear phagocyte
+CL:0000889	myeloid suppressor cell	CL:0000576	Monocytes	Monocytes	CL:0000766	myeloid leukocyte
+CL:0000037	hematopoietic stem cell	CL:0000050	HSC	MEP	CL:0008001	hematopoietic precursor cell
+CL:0000038	erythroid progenitor cell	CL:0000050	HSC	MEP	CL:0008001	hematopoietic precursor cell
+CL:0000084	T cell	CL:0000624	CD4+ T-cells	CD4+ T-cells	CL:0000084	T cell
+CL:0002038	T follicular helper cell	CL:0000624	CD4+ T-cells	CD4+ T-cells	CL:0000624	CD4-positive, alpha-beta T cell
+CL:0000893	thymocyte	CL:0000624	CD4+ T-cells	CD4+ T-cells	CL:0000084	T cell
+CL:0000798	gamma-delta T cell	CL:0000624	CD4+ T-cells	CD4+ T-cells	CL:0000084	T cell
+CL:0000814	NK lymphocyte	CL:0000624	CD4+ T-cells	CD4+ T-cells	CL:0000791	mature alpha-beta T cell
```

Comment:
I have a second here between meetings, so I will return this because it impacts more than one line.

I almost certainly missed this in an earlier review, but this seems like the wrong term for "NK lymphocyte." This seems like a better fit to me: http://purl.obolibrary.org/obo/CL_0000623

Reply:
Ah it looks like the ID is correct, but the name for the ID is wrong in the original file where we assigned IDs. The term from Panglao is "Natural killer T cells" so that should be assigned to [CL:0000814](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCL_0000814), but the name should be "mature NK T cell", not "NK lymphocytes". I'll fix that here.